assembly/assembly-ide-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -37,6 +37,11 @@ project.template_location_dir=${che.home}/templates
 # Folder that contains source code for templates, if you do not want to clone from URL.
 project.template_descriptions_dir=${che.home}/templates
 
+### Configuration of embedded stacks
```

Comment:
why it is not stored as part of the local storage ?
# Che stores various internal data objects as JSON on the file system.
che.conf.storage